### PR TITLE
Feature/wp categories rm

### PR DESCRIPTION
**Pull request by Rochius**
Hi! Here is the file with the categories keywords for the CRUD, also a file with a name generator

### CODE REVIEW

File: wp_api/keywords/categories/categories_keywords.robot
Comment (MartinJAA) [inline wp_api/keywords/categories/categories_keywords.robot:1]: Hi @Rochius, please follow this convention in order to distinguish them:
Keyword: Create New Post
Python method: create_new_post
And add a line at the end of your file, please.

**Review by MartinJAA — CHANGES_REQUESTED** [review]

File: wp_api/resources/imports.robot
Comment (MartinJAA) [inline wp_api/resources/imports.robot:1]: Please fix the indentation, must be the same for all Libraries

File: wp_api/tests/categories/tb78_categories.robot
Comment (MartinJAA) [inline wp_api/tests/categories/tb78_categories.robot:1]: Please add a line a the end of the file

**Review by MartinJAA — CHANGES_REQUESTED** [review]

File: wp_api/resources/imports.robot
Comment (Rochius) [reply wp_api/resources/imports.robot:1]: I already made the changes. Thank you!!

File: wp_api/tests/categories/tb78_categories.robot
Comment (Rochius) [reply wp_api/tests/categories/tb78_categories.robot:1]: I already made the changes. Thank you!!

**Review by MartinJAA — APPROVED** [review]

**Review by Rogerrenjifo — COMMENTED** [review]
Good job my friend Just please remove config.py.

**Review by Rogerrenjifo — APPROVED** [review]
LGTM my friend.

**Pull request merged**